lib/transports/__es__/_base.js

```diff
@@ -42,7 +42,11 @@ class Base {
     }
 
     if (this.parent.options.openSearchServerless) {
-      this.ESversion = 2 // OpenSearch serverless default version
+      // OpenSearch serverless default version
+      // See https://docs.aws.amazon.com/opensearch-service/latest/developerguide/serverless-overview.html
+      // "Currently, serverless collections run OpenSearch version 2.0.x", as of May 13, 2025
+      this.ESversion = 2
+      this.ESFullversion = '2.0.0'
```

Comment:
This is a bad idea, if anything it should be `this.parent.options['force-os-version']`. version 2 disables some features, even though they might not be available in serveless anyway.